pilot/pkg/model/config.go

```diff
@@ -27,8 +27,8 @@ import (
 
 	"github.com/gogo/protobuf/proto"
 
-	mccpb "istio.io/api/mixer/v1/config/client"
 	networking "istio.io/api/networking/v1alpha3"
+	mccpb "istio.io/istio/pilot/pkg/networking/plugin/mixer/client"
```

Comment:
just curious: why the switch away from `istio.io/api` ?

Reply:
Thanks.. Fixed.. leftover from some refactoring in previous pr

Reply:
Changing back because mixer was changed to use that in PR https://github.com/istio/istio/pull/17132/

Reply:
We have to use golang protobuf protos defined by xDS APIs, not the gogo protobuf ones defined by istio.io/api. I'd suggest keeping the vendored golang protos.

Reply:
I'm finding it hard to track the usage, but the general rule is that anything interfacing with Envoy cannot use istio.io/api protos.

Reply:
Keeping the golang proto..